engine/src/main/java/org/destinationsol/game/ship/SolShip.java

```diff
@@ -417,6 +418,9 @@ private void throwLoot(SolGame game, SolItem item, boolean onDeath) {
 
     @Override
     public void receiveDmg(float dmg, SolGame game, Vector2 position, DmgType dmgType) {
+        if(isInvincible == true){
```

Comment:
Please, do not compare booleans to `true`. Ever. It is just ugly. Also, space before brace.
```suggestion
        if (isInvincible) {
```